devtools-ci-index.yaml

```diff
@@ -657,6 +657,78 @@
             exit $rtn_code
 
 - job:
+    name: 'devtools-che-test-end-to-end'
+    defaults: global
+    node: devtools
+    properties:
+        - github:
+            url: https://github.com/redhat-developer/che-e2e-tests
+    scm:
+        - git:
+            url: https://github.com/redhat-developer/che-e2e-tests.git
+
+            shallow_clone: true
+            branches:
+                - master
+    triggers:
+        - github
+        - github-pull-request
+            white-list:
+                - mlabuda
+            trigger-phase: '[testPR]'
```

Comment:
maybe we should use the same trigger phrase as the other projects, so we dont have a different one for each repo ?